### PR TITLE
Validation: Updated ms.author field and H1s.

### PR DESCRIPTION
**Pull request by dereklh7**
Updated ms.author field, added missing H1s, and added missing meta descriptions.

### CODE REVIEW

**Comment by dereklh7** [comment]
@xuzhg

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: Odata-docs/odatalib/v7/add-annotations-to-enummember.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Add vocabulary annotations to EdmEnumMember in odatalib v7"
-description: ""
+description: "Describes support to add vocabulary annotations to EdmEnumMember."
```
Comment (KenitoInc) [inline Odata-docs/odatalib/v7/add-annotations-to-enummember.md:3]:
```suggestion
description: "Describes how to add vocabulary annotations to EdmEnumMember."
```

File: Odata-docs/odatalib/v7/additional-prefer-header.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Add additional prefer header webapi v7"
-description: ""
+description: "Describes creating request message with preferred header."
```
Comment (KenitoInc) [inline Odata-docs/odatalib/v7/additional-prefer-header.md:3]:
```suggestion
description: "Describes how to create a request message with preferred header."
```

File: Odata-docs/odatalib/v7/aggregation-support.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Basic Uri parser support for aggregations in odatalib"
-description: ""
+description: "Describes basic Uri parser first steps and support."
```
Comment (KenitoInc) [inline Odata-docs/odatalib/v7/aggregation-support.md:3]:
```suggestion
description: "Describes basic Uri parser support for aggregations."
```

File: Odata-docs/odatalib/v7/allow-serialization-of-additional-properties.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Allow serialization of additional properties in odatalib"
-description: ""
+description: "Describes serialization and support for additional properties."
```
Comment (KenitoInc) [inline Odata-docs/odatalib/v7/allow-serialization-of-additional-properties.md:3]:
```suggestion
description: "Describes support for serialization of additional properties."
```

File: Odata-docs/odatalib/v7/alternate-key.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Alternate Key in odatalib"
-description: ""
+description: "Describes support for alternate key and codes to build alternate keys."
```
Comment (KenitoInc) [inline Odata-docs/odatalib/v7/alternate-key.md:3]:
```suggestion
description: "Describes support for alternate keys."
```

**Review by KenitoInc — CHANGES_REQUESTED** [review]

**Comment by dereklh7** [comment]
@KenitoInc I've implemented the requested changes. Please let me know if anything else is needed for this branch to be merged.

**Review by KenitoInc — APPROVED** [review]

File: Odata-docs/odatalib/v6/read-payload-v6.md [hunk 6]
```diff
@@ -1,9 +1,8 @@
 ---
-title: " Read OData Payload"
-description: "Read OData payload using OData Core APIs"
-
+title: "Read OData Payload"
+description: Describes how to read OData payloads using OData Core APIs.
```
Comment (gathogojr) [inline Odata-docs/odatalib/v6/read-payload-v6.md:3]: @dereklh7 Does this mean that the quotation marks are optional?

**Review by gathogojr — APPROVED** [review]
Left a question regarding something I'm curious about other LGTM :shipit:

File: Odata-docs/odatalib/v6/read-payload-v6.md (hunk 6 above)
Comment (dereklh7) [reply Odata-docs/odatalib/v6/read-payload-v6.md:3]: @gathogojr Quotation marks are indeed optional for descriptions and titles.

**Review by xuzhg — APPROVED** [review]

**Pull request merged**